lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3220,15 +3231,22 @@ bool SPIRVToLLVM::translate() {
   return true;
 }
 
+std::string SPIRVToLLVM::transDataLayout(std::string const &GeneralLayout) {
```

Comment:
I would just make it a static function and pass `PM`. Passing `this` is not required but costs the same.